src/main/java/edu/cornell/kfs/fp/CuFPConstants.java

```diff
@@ -21,4 +51,46 @@
     public static final String NON_EMPLOYEE_TRAVEL = "Non-Employee Travel";
 
     public static final String USE_OVERSIZED_CHECK_STUB_TEXT_QUESTION_ID = "UseOversizedCheckStubTextQuestion";
+
+    public static final String ACH_INCOME_FILE_DATE_FORMAT = "yymmddHHmm";
```

Comment:
Is this a valid date format? Should it be "yyMMddHHmm" instead? Sorry for not noticing this earlier.